app/src/main/java/com/peacecorps/pcsa/MainActivityFragment.java

```diff
@@ -46,7 +48,9 @@ public void onClick(View v) {
         safetyResourceButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                startActivity(new Intent(getActivity(), SafetyResources.class));
+                Toast.makeText(getActivity(), "This functionality is currently unavailable.", Toast.LENGTH_SHORT).show();
```

Comment:
This is a screen containing some static data. Content for this screen is pending from PCSA. Do not remove.